packages/camera/camera/ios/Classes/CameraPlugin.m

```diff
@@ -872,6 +872,8 @@ - (void)handleMethodCallAsync:(FlutterMethodCall *)call result:(FlutterResult)re
     NSDictionary *argsMap = call.arguments;
     NSUInteger cameraId = ((NSNumber *)argsMap[@"cameraId"]).unsignedIntegerValue;
     if ([@"initialize" isEqualToString:call.method]) {
+    // TODO: Set actual videoFormat? Or is it possible to set an integer
+    videoFormat = ((NSNumber *)argsMap[@"imageStreamImageFormat"]).unsignedIntegerValue;
```

Comment:
This will indeed require a conversion. The supplied value from Dart needs to be translated to one of these formats: https://developer.apple.com/documentation/corevideo/1563591-pixel_format_identifiers?language=objc